visualqa/trainMLP.py

```diff
@@ -38,10 +42,14 @@ def load_train_data(data_root):
 
 def parse_arguments():
     parser = argparse.ArgumentParser()
-    parser.add_argument('-num_hidden_units', type=int, default=1024)
-    parser.add_argument('-num_hidden_layers', type=int, default=3)
+    parser.add_argument('-num_hidden_units', type=int, default=1024,
+                        help="Hidden layer size in the final MLP.")
+    parser.add_argument('-num_hidden_layers', type=int, default=3,
+                        help="Number of hidden layers in the final MLP.")
+    parser.add_argument('-lstm_num_layers', type=int, default=1)
+    parser.add_argument('-lstm_layer_size', type=int, default=256)
     parser.add_argument('-dropout', type=float, default=0.5)
-    parser.add_argument('-activation', type=str, default='tanh')
+    parser.add_argument('-activation', type=str, default='relu')
```

Comment:
Nah, I think it was inherited from the original codebase. I don't expect it to make that big of a difference, but I guess relu is better as a rule of thumb.